static/upload.css

```
@@ -0,0 +1,99 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+    text-align: center;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px 20px;
+    background-color: #f4f4f4;
+}
+
+#logo {
+    max-width: 200px;
+}
+
+.buttons {
+    display: flex;
+    gap: 10px;
+}
```

```suggestion
.buttons {
    display: flex;
    gap: 30px;
    margin-right: 40px;
}
```

Perhaps the buttons can be a bit further from the edge of the screen and with a bit of a wider gap between them?